scripts/Settings.py

```diff
@@ -40,18 +40,23 @@ def layout():
             col1, col2, col3, col4, col5 = st.columns(5, gap='large')
 
             device_list = []
+            selected_device = None
             device_properties = [(i, torch.cuda.get_device_properties(i)) for i in range(torch.cuda.device_count())]
             for device in device_properties:
                 id = device[0]
                 name = device[1].name
                 total_memory = device[1].total_memory
 
                 device_list.append(f"{id}: {name} ({human_readable_size(total_memory, decimal_places=0)})")
+                if str(id) == str(st.session_state.defaults.general.gpu):
+                    selected_device = f"{id}: {name} ({human_readable_size(total_memory, decimal_places=0)})"
+            if selected_device is None:
```

Comment:
That part shows the available memory the GPU has so it's easy to make sure you are selecting the right GPU in case you have multiple devices and with the same names, in my case I have 2 GPUs that are the same model but with different amount of VRAM and this comes handy in that case.